### ISSUE TITLE
[Issue] If Radio is connected to PC while RX is powered/connected, then radio is ejected and disconnected, the receiver will not reconnect ever again on that model.

### ISSUE THREAD

**Issue by BladeScraper-Designs** [FrSkyRC/ETHOS-Feedback-Community#3776]
Currently testing and noticed this on my X20 PRO, with all latest firmwares.  I am unsure if it also occurs on older Ethos versions., but so far, it's been fully repeatable with 1.5.4.  I've only noticed it on my X20 PRO as that is the only radio I'm currently using and it's the only one on 1.5.4, but it may happen on other radios too.

Setup: Receiver (ACCESS or TD, TW untested) powered on and bound/connected, telemetry working
To replicate: With receiver connected, plug radio into PC and select Ethos Suite.  You don't need to do anything, just eject the drives.
Once ejected and you unplug the radio, it will return to Ethos, but the receiver will not reconnect.  Try to rebind, it will not rebind.  Try to register, it will not register.  So far, I've not had any luck in figuring out a way to get the receiver to rebind with the same model.  I always had to create a new model, and it then binds up correctly and connects right away.

This bit me once while I was setting up my new airplane and had to connect to the PC real quick to add some model files and I left the model powered on.  It ruined my model file, and I had to partially start over from an out of date backup.  It didn't occur to me at the time when it happened that it was related to connecting to the PC with the model powered up, but I was doing some unrelated testing with receivers today and it happened again.  As per last time, it would not reconnect, rebind, or re-register.  The only way to solve it was to create a new model.

So far, I've tested and had the same experience with ACCESS (Archer and Archer Plus) and with Tandem.  I do not have any TW receivers easily accessible to test those.

**Comment by hrenz** [comment]
look at your SD-card, there schold be now  copies of your original modells,
take those.

**Comment by strgaltdel** [comment]
Maybe an "artefact" 

it's known that by using suites "dev tools" to switch between firmware and serial mode (for lua developing / debugging purpose) RF section is switched off.
"RF module" on ethos UI remains on state "switched on", but you'll have to reboot your tx to recover transmission

**Comment by BladeScraper-Designs** [comment]
Ah, so it is because of that.  So it seems this is related to/the same root problem as #3753 .  I could have sworn that I tried rebooting when it happened to me the first time and it still would not reconnect, but alas I just tried it and it reconnected after a power cycle.  Now I feel a bit silly for wasting my time creating a new model when it happened to me originally.  Lol.  Still, it seems like the RF module is getting accidentally stuck in an off state after disconnecting from PC, so yes, looks like a bug.
